tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker-test.php

```
@@ -576,6 +628,102 @@ function () {
+		try {
+			$this->sut->sync_inventory();
+		} finally {
+			remove_filter( 'wc_stripe_agentic_commerce_should_sync_product', $callback, 10 );
+		}
+
+		$pending = get_option( WC_Stripe_Agentic_Commerce_Inventory_Tracker::PENDING_UPDATES_OPTION, [] );
+		$this->assertEmpty( $pending, 'Both entries should be cleared — kept_product was uploaded, flipped_product was pruned.' );
+
+		// The uploaded CSV must not contain the flipped product's id.
+		$this->assertNotNull( $uploaded_feed, 'A feed should have been uploaded.' );
+	}
```

PHPMD (2.15.0)

[error] The method test_sync_inventory_prunes_entries_whose_filter_now_excludes is not named in camelCase. (undefined)

(CamelCaseMethodName)